test/static_fixtures/script.txt

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "Calling Ansible AWX/Tower provisioning callback..."
+/usr/bin/curl -v -k -s --data "host_config_key=" https:///api/v2/job_templates//callback/
```

Comment:
This is missing some interpolated values. Can we somehow set up the environment so those are set?

Reply:
Added another test with interpolated values.